packages/just-scripts/src/tasks/webpackTask.ts

```diff
@@ -99,7 +104,8 @@ export function webpackDevServerTask(options: WebpackTaskOptions = {}) {
   return function webpackDevServer() {
     if (devServerCmd && configPath && fs.existsSync(configPath)) {
       const mode = options.mode || 'development';
-      const args = [...(options.nodeArgs || []), devServerCmd, '--config', configPath, '--open', '--mode', mode];
+      const open = options.noOpen ? '' : '--open';
```

Comment:
I know it's a default behavior change, but it is one I would accept - could you add a flag to "open" rather than "noOpen". I am slowly migrating people away from having these flags automatically added for them (like you have discovered here)